spec.html

```diff
@@ -12670,6 +12675,26 @@ <h1>Runtime Semantics: ArgumentListEvaluation</h1>
       </emu-clause>
     </emu-clause>
 
+    <emu-clause id="sec-import-calls">
+      <h1><ins>Import Calls</ins></h1>
+
+      <emu-clause id="sec-import-call-runtime-semantics-evaluation">
+        <h1>Runtime Semantics: Evaluation</h1>
+
+        <emu-grammar>ImportCall : `import` `(` AssignmentExpression `)`</emu-grammar>
+        <emu-alg>
+          1. Let _referencingScriptOrModule_ be ! GetActiveScriptOrModule().
+          1. Let _argRef_ be the result of evaluating |AssignmentExpression|.
+          1. Let _specifier_ be ? GetValue(_argRef_).
```

Comment:
Can you clarify when this error would occur?

Reply:
Like if you refer to an undefined variable?